ModularTegustation/tegu_items/lc13unique_items.dm

```diff
@@ -0,0 +1,499 @@
+	//Defective Manager Bullet PLACEHOLDER OR PROTOTYPE SHIELDS
+/obj/item/managerbullet
+	name = "prototype manager bullet"
+	desc = "Welfare put out a notice that they lost a bunch of manager bullets. This must be one of them."
+	icon = 'icons/obj/ammo.dmi'
+	icon_state = "sleeper-live"
+	var/bullettype = 1
```

Comment:
You should've probably made them all a different item path instead of using this hack.

Reply:
(You actually did...)
Then, this all is not needed. Simply add a proc responsible for bullet effect and modify it for each bullet type.